eq-author/src/App/page/Design/Validation/AnswerValidation.js

```diff
@@ -57,7 +57,13 @@ const formatValue = (value, { type, properties }) => {
     return `£${value}`;
   }
   if (type === UNIT) {
-    return `${value} ${unitConversion[properties.unit].abbreviation}`;
+    let valueWithUnit;
+    unitConversion[properties.unit] == null
+      ? (valueWithUnit = `${value}`)
+      : (valueWithUnit = `${value} ${
+          unitConversion[properties.unit].abbreviation
+        }`);
+    return valueWithUnit;
```

Comment:
Hey Sam, tested and working; looks good. Just got a suggestion for cutting down the code:

```suggestion
    return `${value} ${unitConversion[properties.unit]?.abbreviation || ""}`;
```

Gave it a whirl and it works the same. Also just a heads up - when an object's attribute can't be found it'll return `undefined` instead of `null`. Your comparison works because you've used `==` and `null == undefined`.